### PR TITLE
[abseil] Add uwp support

### PR DESCRIPTION
**Pull request by JackBoosY**
Use [#594](https://github.com/abseil/abseil-cpp/pull/594) changes and add some other changes to fix uwp support.

Since upstream fixed the usage error, removed patch `fix-usage-lnk-error.patch` and tested successfully.

Related: #10254 #9971.

This port doesn't contain any features.

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
LGTM

**Comment by PhoebeHui** [comment]
@strega-nil, could you help review?

**Comment by JackBoosY** [comment]
`grpc` will update to 1.27.3 in #10242, but `abseil` became an optional dependency in the [pre2-release version](https://github.com/grpc/grpc/releases/tag/v1.28.0-pre2).

So, when `grpc` is updated to the next official release, I will change abseil to the feature of `grpc` in another update PR.

@strega-nil Could you please review this PR?

Thanks.

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -45,7 +45,7 @@
 7zip:x64-osx=fail
 7zip:x64-uwp=fail
 abseil:arm-uwp=fail
-abseil:x64-uwp=fail
+abseil:arm64-windows=fail
```
Comment (strega-nil) [inline scripts/ci.baseline.txt]: Why doesn't this support arm64-windows anymore? I don't think I understand

**Review by strega-nil — COMMENTED** [review]
Mostly looks good, but I'm confused about the one thing :)

File: scripts/ci.baseline.txt (hunk 1 above)
Comment (JackBoosY) [reply scripts/ci.baseline.txt]: I think it's useless.

**Comment by strega-nil** [comment]
Alright, awesome! Thanks so much @JackBoosY :)

File: ports/opencensus-cpp/CONTROL [hunk 2]
```diff
@@ -1,8 +1,9 @@
 Source: opencensus-cpp
-Version: 0.4.0
+Version: 0.4.0-1
 Homepage: https://github.com/census-instrumentation/opencensus-cpp
 Description: OpenCensus is a toolkit for collecting application performance and behavior data. It currently includes an API for tracing and stats.
 Build-Depends: abseil
+Supports: !windows
```
Comment (strega-nil) [inline ports/opencensus-cpp/CONTROL]: Is there a reason for this now?

Comment (strega-nil) [reply ports/opencensus-cpp/CONTROL]: cools

**Comment by strega-nil** [comment]
Alright, this looks good to me ! :D

**Pull request merged**